L/libsingular_julia_jll/Compat.toml

```diff
@@ -21,7 +21,7 @@ julia = "1.5.0-1"
 
 ["0.5-0"]
 JLLWrappers = "1.2.0-1"
-Singular_jll = "402.0.1"
+Singular_jll = "402.0.1-402.0"
```

Comment:
@giordano As you know, the goal here is to replicate the effect of a Project.toml compat entry of the form `~402.0.1`. I believe it is, based on my believe that the string here is basically a valid argument for the `VersionSpec` constructor, and I see this:
```
julia> VersionSpec("402.0.1-402.0") == Pkg.Types.semver_spec("~402.0.1")
true
```
Perhaps somebody who really knows can confirm this is the correct way to do it, e.g. @fredrikekre ?